backend/estimate-service/src/main/resources/application.properties

```diff
@@ -40,8 +40,8 @@ egov.idgen.host=https://works-dev.digit.org
 egov.idgen.path=/egov-idgen/id/_generate
 egov.idgen.estimate.number.name=estimate.number
 egov.idgen.estimate.number.format=EP/[fy:yyyy-yy]/[cy:MM]/SEQ_ESTIMATE_NUM_[d{4}]
-egov.idgen.sub.estimate.number.name=sub.estimate.number
-egov.idgen.sub.estimate.number.format=EP/[fy:yyyy-yy]/[cy:MM]/SEQ_ESTIMATE_NUM_[d{4}]/SEQ_SUB_ESTIMATE_NUM
+egov.idgen.estimate.detail.number.name=estimateDetail.number
```

Comment:
The value of `idname` is also not in camel case but lowercase with dot separator. Follow the pattern.